Gruntfile.coffee

```diff
@@ -33,25 +33,29 @@ module.exports = (grunt) ->
           'watch'
         ]
 
-    # Compass configuration
-    compass:
-      options:
-        require: [
-          'compass_sumo'
+    # Sass configuration
+    sass:
+      dev:
```

Comment:
Voor options is dit inderdaad het geval, maar files kunnen momenteel niet op een propere manier gedeeld worden over targets. Ik zou de files-configuratie wel bovenaan kunnen zetten boven de initConfig, maar dan geraakt de configuratie overal verspreid.